app/views/posts/index.html.erb

```diff
@@ -20,8 +20,12 @@
       <% post.recent_comments.each do |comment| %>
         <p><%=comment.author.name%>: <%=comment.text%></p>
       <% end %>
-    
+     
+      
```

Comment:
- [ ] As I can't select the concerned part I select just this area, please the title of the post should show and not a title that was already given by you with the post id, for example, if my post title is `Ruby on Rails` so the title should be `Ruby on Rails` and not `Post #1`. Kindly fix that 🙏🏾 

<img width="633" alt="Screenshot 2023-06-17 at 19 04 42" src="https://github.com/Newtayo/blog_app/assets/74501523/7a7ce55c-a3c7-4be2-89b6-333bb86311ab">